chainlink-testing-framework/mask-toml-secrets/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Mask TOML Secrets'
+description: 'Masks all secret values in Base64-encoded TOML configurations within GitHub Actions logs'
+inputs:
+  base64_toml_content:
+    description: 'Base64-encoded TOML configurations separated by spaces that contain "_secret" keys to be masked'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - run: pip install toml
+      shell: bash
+
+    - name: Run mask script tests
+      run: python3 ${{ github.action_path }}/mask_toml_secrets.py --test
+      shell: bash
```

Comment:
any reason for always running tests, when we call this action? better safe than sorry?

Reply:
@Tofel exactly. And it's less than 100ms I think